tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```
@@ -141,12 +141,14 @@ spec:
 
           if [ "$(jq -r '.state' <<< "${build}")" = "complete" ]; then
             indexImageResolved="$(jq -r '.index_image_resolved' <<<  "${build}")"
-            newCatalogCreatedDate="$(date --date "$(skopeo inspect --config "docker://${indexImageResolved}" | \
-              jq -r .created)" "+%s")"
+            newCatalogCreatedDate="$(date --date "$(skopeo inspect \
+            --retry-times 3 --config "docker://${indexImageResolved}" | \
+            jq -r .created)" "+%s")"
```

Nit: Indent for better readability.
```suggestion
              --retry-times 3 --config "docker://${indexImageResolved}" | \
              jq -r .created)" "+%s")"
```